superset/assets/src/dashboard/stylesheets/variables.less

```diff
@@ -16,62 +16,45 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-@indicator-color: #44C0FF;
+@indicator-color: #44c0ff;
```

Comment:
not complaining, but wondering what autoformatter fixed this. prettier?

Reply:
I'm pretty sure it's prettier + vscode format on save. Happy to turn it off and revert these changes, however I saw there has been discussion about getting prettier into the FE codebase, so reverting may be a wasted effort.

Reply:
No need to revert! Was only curious